spec/fixtures/files/gias_response/ecf_tech.csv

```diff
@@ -1,7 +1,7 @@
 URN,LA (code),LA (name),EstablishmentNumber,EstablishmentName,TypeOfEstablishment (code),TypeOfEstablishment (name),EstablishmentStatus (code),EstablishmentStatus (name),OpenDate,CloseDate,PhaseOfEducation (code),PhaseOfEducation (name),UKPRN,LastChangedDate,Street,Locality,Address3,Town,Postcode,SchoolWebsite,MainEmail,AlternativeEmail,Section41Approved (name),DistrictAdministrative (code),DistrictAdministrative (name),Easting,Northing,PreviousLA (code),PreviousLA (name),OfstedRating (code),OfstedRating (name)
-20001,873,Cambridgeshire,,The Starship Children's Centre,47,Children's centre,1,Open,18/12/2009,,0,Not applicable,,29/11/2019,Bar Hill Primary School,,Gladeside,Bar Hill,CB23 8DY,,1@example.com,2@example.com,Not Approved,E07000012,South Cambridgeshire,538104,263602,999,,99,
-20002,909,Cumbria,,Carlisle South Petteril Bank Upperby SureStart Children's Centre,47,Children's centre,1,Open,21/01/2005,,0,Not applicable,,29/11/2019,Petteril Bank School,Burnett Road,,Carlisle,CA1 3BX,,3@example.com,4@example.com,Not Applicable,E07000028,Carlisle,341828,553461,999,,99,
-20003,852,Southampton,,St Thomas � Becket Church of England Aided Primary School,47,Children's centre,1,Open,28/03/2006,,0,Not applicable,,24/08/2017,"Clovelly,  60-68 Clovelly Road",Central,,Southampton,SO14 0AU,http://sid.southampton.gov.uk/kb5/southampton/directory/family.page?familychannel=2,5@example.com,6@example.com,Not Applicable,E06000045,Southampton,442601,112587,999,,99,
+20001,873,Cambridgeshire,,The Starship Children's Centre,57,Children's centre,1,Open,18/12/2009,,0,Not applicable,,29/11/2019,Bar Hill Primary School,,Gladeside,Bar Hill,CB23 8DY,,1@example.com,2@example.com,Not Approved,E07000012,South Cambridgeshire,538104,263602,999,,99,
+20002,909,Cumbria,,Carlisle South Petteril Bank Upperby SureStart Children's Centre,57,Children's centre,1,Open,21/01/2005,,0,Not applicable,,29/11/2019,Petteril Bank School,Burnett Road,,Carlisle,CA1 3BX,,3@example.com,4@example.com,Not Applicable,E07000028,Carlisle,341828,553461,999,,99,
+20003,852,Southampton,,St Thomas � Becket Church of England Aided Primary School,57,Children's centre,1,Open,28/03/2006,,0,Not applicable,,24/08/2017,"Clovelly,  60-68 Clovelly Road",Central,,Southampton,SO14 0AU,http://sid.southampton.gov.uk/kb5/southampton/directory/family.page?familychannel=2,5@example.com,6@example.com,Not Applicable,E06000045,Southampton,442601,112587,999,,99,
```

Comment:
Codes `47` and `48` no longer appear at all in the GIAS export so I removed them from the mapping (as I had no idea what the keys should be!)